packages/components/container/src/main.ts

```diff
@@ -0,0 +1,3 @@
+import type Main from './main.vue'
+
+export type MainInstance = InstanceType<typeof Main>
```

Comment:
It might be better to move the contents of this file to 'container/index.ts'

And can you refactor the `container.vue`, `aside.vue` and other files at the same time?

Reply:
Ok~